src/home/views/index.tsx

```diff
@@ -1,64 +1,69 @@
+import { channelsListUrl } from "@saleor/channels/urls";
 import useAppChannel from "@saleor/components/AppLayout/AppChannelContext";
 import useNavigator from "@saleor/hooks/useNavigator";
 import useUser from "@saleor/hooks/useUser";
 import React from "react";
 
-import { getUserName, maybe } from "../../misc";
+import { getUserName } from "../../misc";
 import { orderListUrl } from "../../orders/urls";
 import { productListUrl, productVariantEditUrl } from "../../products/urls";
 import { OrderStatusFilter, StockAvailability } from "../../types/globalTypes";
 import HomePage from "../components/HomePage";
-import { HomePageQuery } from "../queries";
+import { useHomePage } from "../queries";
 
 const HomeSection = () => {
   const navigate = useNavigator();
   const { user } = useUser();
   const { channel } = useAppChannel();
 
+  const noChannel = !channel && typeof channel !== "undefined";
+
+  const { data } = useHomePage({
+    displayLoader: true,
+    variables: { channel: channel?.slug },
+    skip: false
```

Comment:
I think query is not skipped by default, so passing `skip: false` seems to be redundant. If the query shouldn't be made, when the channel isn't available, can it be `skip: noChannel`?

Reply:
Ah yes, sorry. I forgot to change it after testing